non-critical-infra/modules/mailserver/default.nix

```diff
@@ -41,6 +47,26 @@
     path = "${config.mailserver.dkimKeyDirectory}/nixos.org.mail.key";
   };
 
+  sops.secrets."nixos.org.${arc.selector}.key" = {
+    format = "binary";
+    owner = "rspamd";
+    group = "rpsamd";
+    mode = "0400";
+    # rspamadm dkim_keygen --selector arc-2025 --domain nixos.org --type rsa --bits 2048
```

Comment:
nit: Could you elaborate on this a bit? Specifically where the public half of this comes from? See https://github.com/NixOS/infra/blob/d30eea236dc043be2b9c0852f17876fef65132e0/non-critical-infra/modules/mailserver/default.nix#L24-L36 for how I handled this for DKIM.